### PR TITLE
Revise content of tours due to new start menu

### PR DESCRIPTION
**Pull request by thinkh**
Fixes https://github.com/Caleydo/tdp_bi_bioinfodb/issues/1243

Requires https://github.com/phovea/phovea_ui/pull/154

### Summary 

* Change tours according to feedback in issue

@oltionchampari Please check carefully if all the points (steps, content, highlight, etc.) have been adressed correctly. Thanks.

### CODE REVIEW

File: src/tours/StartMenuTour.ts [hunk 1]
```diff
@@ -8,76 +8,107 @@ export class StartMenuTour {
...
           Use the "Next" button or the "Return" key to iterate through all the steps. You can use the
           <i>"Cancel"</i> button at any time to stop the tour and to interact with Ordino.
           Please note that the tour will load a new analysis session and the current
           one will be discarded.
-        </p>`,
+        </p>`
+      },
+
+      {
+        selector: 'ul[data-header="mainMenu"]',
```
Comment (oltionchampari) [inline src/tours/StartMenuTour.ts:20]: This point is missing: highlight all 3 tabs ("Datasets, Analysis Session, Tours")
The main menu is not highlighted for me

**Review by oltionchampari — CHANGES_REQUESTED** [review]
I changed Public to Other on the StartMenuTour

Two issues
1. The Add score tour has scrollbars now on the x and y axis

**Comment by thinkh** [comment]
> The Add score tour has scrollbars now on the x and y axis
What do you mean by this? Which step do you get the scrollbars?

Can you please test it in conjunction with PR https://github.com/datavisyn/tdp_core/pull/516 and report again? Thanks.

File: src/tours/StartMenuTour.ts (hunk 1 above)
Comment (thinkh) [reply src/tours/StartMenuTour.ts:20]: Yes, the highlight is not so easy. I will see if I can find a solution.

Comment (thinkh) [reply src/tours/StartMenuTour.ts:20]: Fixed with b1ec4f1bb99480fa4a3df413106461929a6e0399.

grafik

**Comment by oltionchampari** [comment]
> > The Add score tour has scrollbars now on the x and y axis
> 
> What do you mean by this? Which step do you get the scrollbars?
> 
> Can you please test it in conjunction with PR [datavisyn/tdp_core#516](https://github.com/datavisyn/tdp_core/pull/516) and report again? Thanks.

I meant this step

grafik

The PR  datavisyn/tdp_core#516 seems to fix the issue :+1:

**Review by oltionchampari — APPROVED** [review]
Everything works now

**Pull request merged**